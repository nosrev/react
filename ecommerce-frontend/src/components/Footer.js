import React from 'react';
import logoFooter from '../img/logo-footer.png';
import payment from '../img/payment.png';
import { Link } from 'react-router-dom';
import { ReactComponent as Facebook } from '../svg/facebook.svg';
import { ReactComponent as Twitter } from '../svg/twitter.svg';
import { ReactComponent as Behance } from '../svg/behance.svg';
import { ReactComponent as Dribble } from '../svg/dribble.svg';
import { ReactComponent as Linkedin } from '../svg/linkedin.svg';

const Footer = () => {
    return (
        <div id="footer">
            <div className="container mx-auto">
                <div className="flex flex-wrap md:flex-nowrap">
                    <div className="basis-full md:basis-1/4">
                        <Link to='/'>
                            <img src={logoFooter} alt="Store" className="mx-auto" />
                        </Link>
                        <ul className="socials">
                            <li>
                                <a href="https://facebook.com" target="_blank" rel="noreferrer">
                                    <Facebook className="icon w-5 fill-slate-300" />
                                </a>
                            </li>
                            <li>
                                <a href="https://twitter.com" target="_blank" rel="noreferrer">
                                    <Twitter className="icon w-5 fill-slate-300" />
                                </a>
                            </li>
                            <li>
                                <a href="https://www.behance.net" target="_blank" rel="noreferrer">
                                    <Behance className="icon w-5 fill-slate-300" />
                                </a>
                            </li>
                            <li>
                                <a href="https://dribbble.com" target="_blank" rel="noreferrer">
                                    <Dribble className="icon w-5 fill-slate-300" />
                                </a>
                            </li>
                            <li>
                                <a href="https://linkedin.com" target="_blank" rel="noreferrer">
                                    <Linkedin className="icon w-5 fill-slate-300" />
                                </a>
                            </li>
                        </ul>
                    </div>
                    <div className="basis-full md:basis-2/4">
                        <ul className="menu">
                            <li>
                                <Link to='/shop'>
                                    Homem
                                </Link>
                            </li>
                            <li>
                                <Link to='/shop'>
                                    Mulher
                                </Link>
                            </li>
                            <li>
                                <a href="/">Lookbook</a>
                            </li>
                            <li>
                                <a href="/">Oferta</a>
                            </li>
                            <li>
                                <a href="/">Blog</a>
                            </li>
                        </ul>
                        <div className="newsletter">
                            <h6>Inscreva-se</h6>
                            <input type="text" className="input" placeholder="Email Address" required />
                            <button className="btn-submit">
                                Enviar
                            </button>
                        </div>
                    </div>
                    <div className="basis-full md:basis-1/4">
                        <h6>Contato</h6>
                        <p className="contact">
                            Av Paulista, 100000<br />
                            Consolação - São Paulo<br />
                            (11) 9999-9999
                        </p>
                        <img src={payment} alt="" />
                    </div>
                </div>
            </div>
        </div>
    )
}

export default Footer;
