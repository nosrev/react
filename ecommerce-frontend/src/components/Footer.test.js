import renderer from 'react-test-renderer';
import Footer from './Footer';
import { MemoryRouter as Router } from 'react-router-dom';

it('render Footer', () => {
  const component = renderer.create(
    <Router>
        <Footer />
    </Router>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
