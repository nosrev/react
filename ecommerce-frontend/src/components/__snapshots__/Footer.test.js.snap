// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Footer 1`] = `
<div
  id="footer"
>
  <div
    className="container mx-auto"
  >
    <div
      className="flex flex-wrap md:flex-nowrap"
    >
      <div
        className="basis-full md:basis-1/4"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <img
            alt="Store"
            className="mx-auto"
            src="logo-footer.png"
          />
        </a>
        <ul
          className="socials"
        >
          <li>
            <a
              href="https://facebook.com"
              rel="noreferrer"
              target="_blank"
            >
              <svg
                className="icon w-5 fill-slate-300"
              >
                facebook.svg
              </svg>
            </a>
          </li>
          <li>
            <a
              href="https://twitter.com"
              rel="noreferrer"
              target="_blank"
            >
              <svg
                className="icon w-5 fill-slate-300"
              >
                twitter.svg
              </svg>
            </a>
          </li>
          <li>
            <a
              href="https://www.behance.net"
              rel="noreferrer"
              target="_blank"
            >
              <svg
                className="icon w-5 fill-slate-300"
              >
                behance.svg
              </svg>
            </a>
          </li>
          <li>
            <a
              href="https://dribbble.com"
              rel="noreferrer"
              target="_blank"
            >
              <svg
                className="icon w-5 fill-slate-300"
              >
                dribble.svg
              </svg>
            </a>
          </li>
          <li>
            <a
              href="https://linkedin.com"
              rel="noreferrer"
              target="_blank"
            >
              <svg
                className="icon w-5 fill-slate-300"
              >
                linkedin.svg
              </svg>
            </a>
          </li>
        </ul>
      </div>
      <div
        className="basis-full md:basis-2/4"
      >
        <ul
          className="menu"
        >
          <li>
            <a
              href="/shop"
              onClick={[Function]}
            >
              Homem
            </a>
          </li>
          <li>
            <a
              href="/shop"
              onClick={[Function]}
            >
              Mulher
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              Lookbook
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              Oferta
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              Blog
            </a>
          </li>
        </ul>
        <div
          className="newsletter"
        >
          <h6>
            Inscreva-se
          </h6>
          <input
            className="input"
            placeholder="Email Address"
            required={true}
            type="text"
          />
          <button
            className="btn-submit"
          >
            Enviar
          </button>
        </div>
      </div>
      <div
        className="basis-full md:basis-1/4"
      >
        <h6>
          Contato
        </h6>
        <p
          className="contact"
        >
          Av Paulista, 100000
          <br />
          Consolação - São Paulo
          <br />
          (11) 9999-9999
        </p>
        <img
          alt=""
          src="payment.png"
        />
      </div>
    </div>
  </div>
</div>
`;
